<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #181818;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>271</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #3b2b5c;
border-top-right-radius: 20px;
border-bottom-right-radius: 20px;
border-bottom-left-radius: 3px;
border-top-left-radius: 3px;</string>
    </property>
    <widget class="QPushButton" name="addTask">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #897a99;
border-radius: 10px
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="boxShowOn" native="true">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #897a99;
border-radius: 15px</string>
     </property>
     <widget class="QPushButton" name="showOn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  #5f3dc4;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoRepeatDelay">
       <number>0</number>
      </property>
      <property name="autoRepeatInterval">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="boxGrade" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #897a99;
border-radius: 15px</string>
     </property>
     <widget class="QPushButton" name="gradeTask">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  #087f5b;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="inTask">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #897a99;
border-radius: 10px; 
font-size: 16px;
color: #343a40;
</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scroll">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>261</width>
       <height>481</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>481</height>
       </rect>
      </property>
     </widget>
    </widget>
    <zorder>boxShowOn</zorder>
    <zorder>boxGrade</zorder>
    <zorder>addTask</zorder>
    <zorder>inTask</zorder>
    <zorder>scroll</zorder>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>20</y>
      <width>61</width>
      <height>561</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  #3b2b5c;
border-top-left-radius: 20px;
border-bottom-left-radius:20px;
border-top-right-radius: 2px;
border-bottom-right-radius: 2px;</string>
    </property>
    <widget class="QWidget" name="boxShowOff" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #897a99;
border-radius: 15px</string>
     </property>
     <widget class="QPushButton" name="showOff">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  #0b7285;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoRepeatDelay">
       <number>0</number>
      </property>
      <property name="autoRepeatInterval">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="boxExit" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>510</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #897a99;
border-radius: 15px</string>
     </property>
     <widget class="QPushButton" name="btnExit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:  #631D1D;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoRepeatDelay">
       <number>0</number>
      </property>
      <property name="autoRepeatInterval">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="h2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>40</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #897a99;
border-radius: 10px</string>
     </property>
     <property name="text">
      <string> &lt; h2 &gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="h1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>40</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #897a99;
border-radius: 10px</string>
     </property>
     <property name="text">
      <string>&lt; h1 &gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="h3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>40</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #897a99;
border-radius: 10px</string>
     </property>
     <property name="text">
      <string>&lt; h3 &gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>40</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  #897a99;
border-radius: 10px</string>
     </property>
     <property name="text">
      <string>&lt; p &gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>611</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #3d354d;
border-radius: 20px;</string>
    </property>
    <widget class="QPlainTextEdit" name="inBook">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>571</width>
       <height>501</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;
font-size: 15px;</string>
     </property>
    </widget>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>widget_4</zorder>
   <zorder>widget</zorder>
  </widget>
  <action name="addTask_2">
   <property name="text">
    <string>addTask</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionqwe">
   <property name="text">
    <string>qwe</string>
   </property>
  </action>
  <action name="actionasdasd">
   <property name="text">
    <string>asdasd</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
